mitodl/open-discussions
# First pass on video drawer

#### Pre-Flight checklist


#### What are the relevant tickets?
(Required)

#### What's this PR do?
(Required)

#### How should this be manually tested?
- Reimport youtube videos (`course_catalog.etl.pipelines.youtube_etl()`)
- Go to search page and verify the drawer works and data displays as expected (e.g. duration, etc)

#### Screenshots (if appropriate)

Desktop:
![Screenshot_2019-11-08 MIT Open Learning](https://user-images.githubusercontent.com/28598/68512847-79f11e00-0247-11ea-98d6-810e194c2e08.png)

Mobile:
![Screenshot_2019-11-08 MIT Open Learning(2)](https://user-images.githubusercontent.com/28598/68513043-026fbe80-0248-11ea-908c-b19c03f74450.png)
